maintenance/checkLastWikiActivity.php

```diff
@@ -42,18 +42,20 @@ public function execute(): void {
 	}
 
 	private function getTimestamp(): int {
+		$default_id = $this->getServiceContainer->getUserFactory->newFromName("MediaWiki default")->getActorId()
 		$dbr = $this->getReplicaDB();
 		$timestamp = $dbr->newSelectQueryBuilder()
 			->select( 'MAX(rc_timestamp)' )
 			->from( 'recentchanges' )
 			->where( [
 				$dbr->expr( 'rc_log_type', '!=', 'renameuser' ),
 				$dbr->expr( 'rc_log_type', '!=', 'newusers' ),
+				$dbr->expr( 'rc_actor', '!=', $default_id) 
```

Comment:
Missing space before `)` and missing trailing comma

Reply:
Is a trailing comment on the last line of a list really needed? Regardless, added(these are on local, resolving to keep track before committing)

Reply:
Technically not needed, but it's MediaWiki convention I think (and definitely Python)